fineract-provider/src/main/java/org/apache/fineract/infrastructure/security/filter/TwoFactorAuthenticationFilter.java

```diff
@@ -113,7 +113,7 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
         chain.doFilter(req, res);
     }
 
-    @SuppressWarnings("deprecation") // TODO FINERACT-1012
+    // FINERACT-1012
```

Comment:
I think the deprecation suppression here is directly related to FINERACT-1012. Unless this resolves them somehow, I think it should remain.